<ui version="4.0" >
 <class>AudioConvClass</class>
 <widget class="QDialog" name="AudioConvClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>AudioConv</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QPushButton" name="mLoadBtn" >
       <property name="text" >
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mConvertBtn" >
       <property name="text" >
        <string>Convert</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="mLogText" />
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
